index.html

```diff
@@ -16,22 +16,37 @@
 
 <body>
 
- <!-- navigation code here -->
+ <!-- navigation code here???? Why here? -->
+  <header class="main-top">
+    <h1>Space Walkers</h1>
+    <nav>
+        <a href="">Home</a>
+        <a href="">About Us</a>
+        <a href="">Blog</a>
+        <a href="">Contact</a>
+        <a href="">Sign In</a>
+    </nav>
+  </header>
 
   <div class="container home">
 
-    <header class="intro">
-       <!-- image code here -->
+    <div class="intro">
+  
+      <img src="img/main-header.png" class="top-image" alt="">
```

Comment:
This image doesn't shrink when in mobile mode. Maybe try giving this a width of 100% and shrinking the container it's in when in mobile mode. That was the only flaw I saw in this. Otherwise it was great!